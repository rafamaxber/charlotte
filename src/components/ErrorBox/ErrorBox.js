// TODO: Implement a generic error box

import React, { Component } from 'react';
import styled from 'styled-components';

const Stage = styled.div``;
const Box = styled.div``;
const Message = styled.div``;
const CloseBtn = styled.div``;

export default () => (
  <Stage></Stage>
)
