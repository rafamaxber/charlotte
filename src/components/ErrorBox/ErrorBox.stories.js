import React from 'react';
import { storiesOf } from '@storybook/react';
import ErrorBox from './ErrorBox';

storiesOf('ErrorBox', module)
  .add('ErrorBox network error', () => (
    <ErrorBox>
      Network error
    </ErrorBox>
  ));
