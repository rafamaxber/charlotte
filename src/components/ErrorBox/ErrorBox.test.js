import React from 'react';
import renderer from 'react-test-renderer';
import ErrorBox from './ErrorBox';

describe('ErrorBox', () => {
  it('Should match with snapshot', () => {
    const component = renderer.create(
      <ErrorBox></ErrorBox>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
