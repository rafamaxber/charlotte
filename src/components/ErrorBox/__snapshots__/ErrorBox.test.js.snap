// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBox Should match with snapshot 1`] = `
<div
  className="sc-bdVaJa bDWFJH"
/>
`;
